pdg/src/snapshots/c2rust_pdg__tests__analysis_test_pdg_snapshot.snap

```diff
@@ -108,28 +107,26 @@ g {
 	n[12]: field.0    n[3]  => _12 @ bb4[18]: fn simple; _12 = ((*_1).0: i32)                        
 	n[13]: addr.load  n[12] => _   @ bb4[18]: fn simple; _12 = ((*_1).0: i32)                        
 	n[14]: field.1    n[3]  => _13 @ bb4[21]: fn simple; _13 = &raw const ((*_1).1: u64)             
-	n[15]: addr.load  n[14] => _   @ bb4[21]: fn simple; _13 = &raw const ((*_1).1: u64)             
-	n[16]: addr.load  n[3]  => _   @ bb4[24]: fn simple; _14 = &raw const (*_1)                      
```

Comment:
`_14 = &(*_1)` should produce a `Copy` node, shouldn't it?

Reply:
Yes, that was fixed in #533 but this wasn't up to date with master. Now fixed.

Reply:
The `LoadAddr` is gone now, but there's still no `Copy` at `bb4[24]` in the snapshot